packages/core/src/components/logo/logo.scss

```diff
@@ -0,0 +1,66 @@
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/variables/typography";
+
+:host {
+  display: block;
+  font-family: typography.$font-family-base;
```

Comment:
even nieuwsgierig: waarom is dit nodig? ik heb tot nu toe geen `font-family` in een `:host` selector gezien, omdat de `font-family` gewoon zou moeten overerven.

Reply:
Ik zal eens kijken of het zonder ook kan. Ik wilde zeker weten dat het font in de SVG `text` elementen goed werd overgenomen.